File: tests/fixtures/perl/basics.pl
#!/usr/bin/env perl
use strict;
use warnings;
use feature 'say';

my $name = "World";
say "Hello, $name!";

my @items = (1, 2, 3, 4, 5);
my %hash = (key => 'value', foo => 'bar');

sub greet {
    my ($name, $greeting) = @_;
    $greeting //= "Hello";
    return "$greeting, $name!";
}

File: tests/fixtures/lua/basics.lua
local function greet(name)
    name = name or "World"
    print("Hello, " .. name .. "!")
end

local items = {1, 2, 3, 4, 5}
local config = {
    host = "localhost",
    port = 8080,
    enabled = true
}

for i, v in ipairs(items) do
    print(i, v)
end

for k, v in pairs(config) do
    print(k .. " = " .. tostring(v))
end